content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
```

Comment:
Since you're writing this grant in it, should this be scholarly works?

Reply:
Is works a bit broad? I thinks datasets, jupyter notebooks, and figures all fall under works. Perhaps "scholarly documents"? Were you envisioning changing here or everywhere.

Reply:
I like `scholarly documents` - I agree works is too broad. I think just here is probably ok. I just wondered if this was a point to note that it was used for this grant. 😄

Reply:
Done in https://github.com/greenelab/manufund-2018/pull/2/commits/46ce5137a37b6042e3ff5a740b7f0991cc578b23